<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="0" column="1">
     <widget class="QToolBox" name="tbxMain">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabSpacing">
       <number>2</number>
      </property>
      <widget class="QWidget" name="pgRcvData">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>431</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Приём данных</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="3" column="0" colspan="2">
            <widget class="QLineEdit" name="leDer">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>U2</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLineEdit" name="leMovement">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="leU2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QLineEdit" name="leRes">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="leU1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Перемещение, мм</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QPushButton" name="pushButton_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QPushButton" name="pbtnStart">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Старт</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Результат</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>k∆/∑</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>U1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QPushButton" name="pbtCancel">
             <property name="text">
              <string>Отмена</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pbtWrite">
             <property name="text">
              <string>Записать</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QTableWidget" name="tableWidget">
             <property name="gridStyle">
              <enum>Qt::SolidLine</enum>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>75</number>
             </attribute>
             <row>
              <property name="text">
               <string>1</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>2</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>3</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>4</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>5</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>6</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>7</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>8</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>9</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>10</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>11</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Положение, мм</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>U1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>U2</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>k∆/∑</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Результ.</string>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string>0</string>
              </property>
             </item>
             <item row="1" column="0">
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item row="2" column="0">
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item row="3" column="0">
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item row="4" column="0">
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item row="5" column="0">
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item row="6" column="0">
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item row="7" column="0">
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item row="8" column="0">
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item row="9" column="0">
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item row="10" column="0">
              <property name="text">
               <string>10</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgSettings">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>774</width>
         <height>428</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Настройка</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="gbCOM">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="title">
           <string>Настройка порта</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Скорость (бит/с)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>COM порт</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cmbCOMSpeed">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QComboBox" name="cmbCOM">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
